test/forge/HalmosTest.sol

```diff
@@ -0,0 +1,151 @@
+            args = abi.encode(marketParams, assets, shares, onBehalf, emptyData);
+        } else if (selector == morpho.withdraw.selector) {
+            args = abi.encode(marketParams, assets, shares, onBehalf, receiver);
+        } else if (selector == morpho.borrow.selector) {
+            args = abi.encode(marketParams, assets, shares, onBehalf, receiver);
+        } else if (selector == morpho.repay.selector) {
+            args = abi.encode(marketParams, assets, shares, onBehalf, emptyData);
+        } else if (selector == morpho.supplyCollateral.selector) {
+            args = abi.encode(marketParams, assets, onBehalf, emptyData);
+        } else if (selector == morpho.withdrawCollateral.selector) {
+            args = abi.encode(marketParams, assets, onBehalf, receiver);
+        } else if (selector == morpho.liquidate.selector) {
+            address borrower = svm.createAddress("borrower");
+            args = abi.encode(marketParams, borrower, assets, shares, emptyData);
+        } else if (selector == morpho.flashLoan.selector) {
+            uint256 rand = svm.createUint256("rand");
+            address token;
+            if (rand == 0) {
+                token = address(loanToken);
+            } else if (rand == 1) {
+                token = address(collateralToken);
+            } else {
+                ERC20Mock otherToken = new ERC20Mock();
+                token = address(otherToken);
+            }
+            args = abi.encode(marketParams, token, assets, emptyData);
+        } else if (selector == morpho.accrueInterest.selector) {
+            args = abi.encode(marketParams);
+        } else if (selector == morpho.setFee.selector) {
+            uint256 newFee = svm.createUint256("newFee");
+            args = abi.encode(marketParams, newFee);
+        } else {
+            args = svm.createBytes(1024, "data");
+        }
+
+        vm.prank(caller);
+        (bool success,) = address(morpho).call(abi.encodePacked(selector, args));
+        vm.assume(success);
+    }
+
+    // Check that the fee is always smaller than the max fee.
+    function check_feeInRange(bytes4 selector, address caller) public {
+        _callMorpho(selector, caller);
+
+        assert(morpho.fee(id) <= MAX_FEE);
+    }
+
+    // Check that there is always less borrow than supply on the market.
+    function check_borrowLessThanSupply(bytes4 selector, address caller) public {
+        _callMorpho(selector, caller);
+
+        assert(morpho.totalBorrowAssets(id) <= morpho.totalSupplyAssets(id));
+    }
+
+    // Check that the market cannot be "destroyed".
+    function check_lastUpdatedNonZero(bytes4 selector, address caller) public {
+        _callMorpho(selector, caller);
+
+        assert(morpho.lastUpdate(id) != 0);
+    }
+
+    // Check that enabled LLTVs are necessarily less than 1.
+    function check_lltvSmallerThanWad(bytes4 selector, address caller, uint256 lltv) public {
+        vm.assume(!morpho.isLltvEnabled(lltv) || lltv < 1e18);
```

Comment:
`lltv` is symbolic in this example, so it could be another one than the one that was defined in the setup. Also the `isLltvEnabled` mapping is symbolic. 
So it does not prevent the case where `!morpho.isLltvEnalbled(lltv)` and `lltv >= 1e18`. Assuming that we call any function that does not touch this `lltv`, for example a view function, then we would still have `!morpho.isLltvEnalbled(lltv)` and `lltv >= 1e18` after the function call and the assert would fail

Reply:
I didn't know that the storage was completely symbolic, that's why I asked this